### PR TITLE
Features/node 10

### PR DESCRIPTION
**Pull request by hynek-urban**
This will help us make builds faster and more robust thanks to using "npm ci" in Travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ sudo: false
 group: beta
 language: node_js
 node_js:
-  - "8"
+  - "10.3.0"
```
Comment (MatiasOS) [inline .travis.yml]: This is not necesary beacuse of the `.nvmrc` file. You can check it at [Travis docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc)

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-8.9.4
+10.3.0
```
Comment (MatiasOS) [inline .nvmrc]: Are we using 10.3 for any particular reason? I ask because the current version is 10.4

File: .travis.yml (hunk 1 above)
Comment (hynek-urban) [reply .travis.yml]: Hi @MatiasOS. Settings in travis.yml trumps what's in .nvmrc, see https://travis-ci.org/windingtree/off-chain-accessor-swarm/builds/388717732 in practice. That's why this had to be changed as well.

File: .nvmrc (hunk 2 above)
Comment (hynek-urban) [reply .nvmrc]: 10.4 came out just a few days ago. When I started working on this last week, 10.3 was the latest one. For the sake of consistency with other repos where 10.3 has been already selected, I would rather keep it at 10.3.

File: .travis.yml (hunk 1 above)
Comment (JirkaChadima) [reply .travis.yml]: I think @MatiasOS means that the whole `node_js` section can be omitted from travis config

File: .nvmrc (hunk 2 above)
Comment (JirkaChadima) [reply .nvmrc]: I'd stick with 10.3 for now. I've already merged Hynek's PRs in some other repositories. We can set up a periodic task in Trello for upgrading node version every so often that someone will always take care of. We're not that dependent on node version for now.

File: .travis.yml (hunk 1 above)
Comment (hynek-urban) [reply .travis.yml]: I did understand that. I didn't put it there, though, that wasn't the point of my change :) I can remove it but then the configuration will diverge among repositories. Anyway, not worth the debate IMO - if you feel strongly about it, feel free to remove the section in question before merging the pull request.

**Pull request merged**